app/src/test/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionPresenter.kt

```diff
@@ -200,6 +200,7 @@ class ProjectSelectionPresenterTest {
         }
 
         val libraryModel = Model("downloaded/0")
+        libraryModel.others.put("benchmark", project.showBenchmark)
```

Comment:
What is this for? Should we create a const for "benchmark" (think this is the second time I see it)?